send_to_datadog.py

```diff
@@ -22,7 +22,7 @@
               'youtube.fx.release', 'youtube.fx.nightly',
               'youtube.chrome.release', 'facebook.fx.release',
               'facebook.fx.nightly', 'facebook.chrome.release']
-metrics = ['TTFB', 'render', 'firstPaint', 'SpeedIndex', 'bytesInDoc',
+metrics = ['TTFB', 'render', 'firstPaint', 'DOMContentFlushed', 'SpeedIndex', 'bytesInDoc',
```

Comment:
Is this case-sensitive? In your other PR it's `domContentFlushed` (lower-case `dom`).